casper/src/test/scala/coop/rchain/casper/api/BlockQueryResponseAPITest.scala

```diff
@@ -94,6 +94,35 @@ class BlockQueryResponseAPITest extends FlatSpec with Matchers with BlockStoreFi
     blockInfo.shardId should be(shardId)
   }
 
+  "findBlockWithDeploy" should "return successful block info response" in withStore {
```

Comment:
It's not clear for me by just looking at this test why it succeeds. There's no explicit process that sets up the underlying block store and/or Casper to actually have the `(ByteString.EMPTY, 1L)` deploy in it. Please make this explicit otherwise in the future someone will be looking at this code pulling his hair trying to figure out why this test passes.

Reply:
By explicit I meant creating a block and then trying to look it up.

Reply:
Yeah that is sort of what I did above.